Vendor.API/Controllers/ServiceController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Vendor.Application.Requests.Services;
+using Vendor.Domain.Entities;
+
+namespace Vendor.API.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]/[action]")]
+    public class ServiceController
+    {
+        private readonly IMediator _mediator;
+        public ServiceController(IMediator mediator) 
```

Comment:
Add summary for all controller - i.t., class, interface, method, constructor. If method specify parameters